### PR TITLE
Fix Scp079Role::Camera NRE

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.API/Features/Camera.cs [hunk 1]
```diff
@@ -233,7 +233,7 @@ public bool IsBeingUsed
         /// </summary>
         /// <param name="camera079">The base <see cref="Scp079Camera"/>.</param>
         /// <returns>A <see cref="Camera"/> or <see langword="null"/> if not found.</returns>
-        public static Camera Get(Scp079Camera camera079) => Camera079ToCamera.TryGetValue(camera079, out Camera camera) ? camera : new(camera079);
+        public static Camera Get(Scp079Camera camera079) => camera079 is not null && Camera079ToCamera.TryGetValue(camera079, out Camera camera) ? camera : new(camera079);
```
Comment (iopietro) [inline Exiled.API/Features/Camera.cs:236]:
```suggestion
        public static Camera Get(Scp079Camera camera079) => camera079 != null && Camera079ToCamera.TryGetValue(camera079, out Camera camera) ? camera : new(camera079);
```

**Review by iopietro — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Roles/Scp079Role.cs [hunk 2]
```diff
@@ -134,7 +135,7 @@ internal Scp079Role(Scp079GameRole baseRole)
         /// </summary>
         public Camera Camera
         {
-            get => Camera.Get(Internal.CurrentCamera);
+            get => Camera.Get(Internal.CurrentCamera) ?? Camera.Get(CameraType.Hcz079ContChamber);
```
[the remark this answers is not in the document]
Comment (louis1706) [reply Exiled.API/Features/Roles/Scp079Role.cs:138]: Done

**Comment by Thundermaker300** [comment]
Fix merge conflict

File: Exiled.API/Features/Roles/Scp079Role.cs [hunk 3]
```diff
@@ -10,7 +10,11 @@ namespace Exiled.API.Features.Roles
     using System.Collections.Generic;
     using System.Linq;
 
+
+    using Exiled.API.Enums;
+
     using Interactables.Interobjects.DoorUtils;
+
     using PlayerRoles;
```
Comment (louis1706) [inline Exiled.API/Features/Roles/Scp079Role.cs:18]:
```suggestion
     using System.Linq;
 
     using Exiled.API.Enums;

     using Interactables.Interobjects.DoorUtils;
     using PlayerRoles;
```

File: Exiled.API/Features/Roles/Scp079Role.cs [hunk 4]
```diff
@@ -153,10 +154,11 @@ internal Scp079Role(Scp079GameRole baseRole)
 
         /// <summary>
         /// Gets or sets the camera SCP-079 is currently controlling.
+        /// <remarks>This value will return the Hcz079ContChamber Camera instead of null</remarks>
```
Comment (Thundermaker300) [inline Exiled.API/Features/Roles/Scp079Role.cs:157]:
```suggestion
        /// <remarks>This value will return the <c>Hcz079ContChamber</c> Camera if SCP-079's current camera cannot be detected.</remarks>
```

File: Exiled.API/Features/Camera.cs [hunk 5]
```diff
@@ -233,7 +233,7 @@ public bool IsBeingUsed
         /// </summary>
         /// <param name="camera079">The base <see cref="Scp079Camera"/>.</param>
         /// <returns>A <see cref="Camera"/> or <see langword="null"/> if not found.</returns>
-        public static Camera Get(Scp079Camera camera079) => Camera079ToCamera.TryGetValue(camera079, out Camera camera) ? camera : new(camera079);
+        public static Camera Get(Scp079Camera camera079) => camera079 != null && Camera079ToCamera.TryGetValue(camera079, out Camera camera) ? camera : new(camera079);
```
Comment (IRacle1) [inline Exiled.API/Features/Camera.cs:236]: Uh, it will try to create camera wrapper when base is null?

Comment (louis1706) [reply Exiled.API/Features/Camera.cs]: skill issue

**Review by iopietro — APPROVED** [review]

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**